sycl/include/sycl/ext/intel/experimental/esimd/simd_view.hpp

```diff
@@ -20,15 +20,15 @@ namespace intel {
 namespace experimental {
 namespace esimd {
 
-/// The reference class.
+/// The simd_view base class.
 ///
 /// This class represents a region applied to a base object, which
 /// must be a simd object.
 ///
```

Comment:
```suggestion
/// The simd_view base class.
///
/// Internal class implementing basic functionality of simd_view.
///
```

Nit: also would be good to add description why the split is needed.

Reply:
Done.